src/lib/components/SnackbarContainer/index.js

```diff
@@ -34,24 +34,51 @@ class SnackbarContainer extends BaseElement {
 
   constructor() {
     super();
+    this.onBeforeInstallPrompt = this.onBeforeInstallPrompt.bind(this);
     this.onStateChanged = this.onStateChanged.bind(this);
   }
 
   connectedCallback() {
     super.connectedCallback();
     store.subscribe(this.onStateChanged);
     this.onStateChanged();
+
+    if (!this.acceptedCookies) {
```

Comment:
I was wondering if it's possible that the user may have already accepted cookies, but hasn't been hit with the install prompt yet. I don't know if it's still the case, but Chrome used to have a heuristic that waited until the user had visited a site more than once before showing them the install prompt. If that's still the case then they may have accepted cookies on the first visit but not seen the prompt yet and this listener would never set up.

Reply:
Well if the user has accepted cookies then we don't need to wait for them to accept cookies and then present it. We only want to prevent the install prompt if they haven't accepted cookies yet. Otherwise let chrome run it's normal business.

Reply:
ah yes, good point!